puppeteer/bundleRequests.test.js

```diff
@@ -15,7 +15,7 @@ let requests = [];
 
 const isJsBundle = url => url.includes(localBaseUrl);
 
-jest.setTimeout(20000); // overriding the default jest timeout
+jest.setTimeout(30000); // overriding the default jest timeout
```

Comment:
Should this change belong in here? If so we should make a note in the PR description as to why it's needed

Reply:
I'll update the description.

I've seen these tests failing more recently, requiring a manual re-run to get them to pass, so I guess the change was made here as it happened on this PR too.